.github/workflows/pylint.yml

```diff
@@ -0,0 +1,22 @@
+name: Pylint
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Shouldn't we use a python image instead of ubuntu like in our containers? Or maybe there is no link?

Reply:
actually we may, but this doesn't change anything, only the python and pylint version matter here because the project isn't launched at all (see the steps) only pylint is
This is just the GH action default setup :wink:

Reply:
Ok perfect, just wanted to be sure :)